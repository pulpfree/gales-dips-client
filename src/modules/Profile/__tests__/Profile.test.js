/* eslint no-undef: "off" */

import React from 'react'
import { shallow } from 'enzyme'

import Profile from '../Profile'

describe('Profile', () => {

  let mountedC
  let props
  const reportC = () => {
    if (!mountedC) {
      mountedC = shallow(
        <Profile {...props} />
      )
    }
    return mountedC
  }

  beforeEach(() => {
    props = {
      history: {location: {pathname: '/'}},
    }
    mountedC = undefined
  })

  it('matches snapshot', () => {
    expect(reportC()).toMatchSnapshot()
  })

})