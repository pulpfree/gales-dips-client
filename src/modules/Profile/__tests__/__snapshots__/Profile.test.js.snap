// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(Profile)
    history={
        Object {
            "location": Object {
              "pathname": "/",
            },
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "classes": Object {
        "additionalButton": "Profile-additionalButton-7",
        "appBar": "Profile-appBar-1",
        "container": "Profile-container-2",
        "formContainer": "Profile-formContainer-4",
        "formControl": "Profile-formControl-5",
        "paper": "Profile-paper-3",
        "submitButton": "Profile-submitButton-6",
      },
      "history": Object {
        "location": Object {
          "pathname": "/",
        },
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "classes": Object {
          "additionalButton": "Profile-additionalButton-7",
          "appBar": "Profile-appBar-1",
          "container": "Profile-container-2",
          "formContainer": "Profile-formContainer-4",
          "formControl": "Profile-formControl-5",
          "paper": "Profile-paper-3",
          "submitButton": "Profile-submitButton-6",
        },
        "history": Object {
          "location": Object {
            "pathname": "/",
          },
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
